PAS-51 Add Application Routes

BraidenCutforth opened the pull request:
- Add react-router-dom dependency
- Add Pages component that contains top level routes.
- Add Pages component to App component
- Add Login and Sign Up page to pages/account subdirectory
- Fix imports in Login and Sign Up
- Adjust application baseUrl for nicer imports

BraidenCutforth committed:
wip

BraidenCutforth committed:
wip

BraidenCutforth committed:
wip

seiyaterada commented on src/pages/index.tsx:
```diff
@@ -0,0 +1,24 @@
+		<Router>
+			<Switch>
+				<Route path={`/sign-in`}>
+					<Login />
```
Maybe keep the names of the route path and the `<Login />` consistent with either sign-in or Login?

BraidenCutforth replied:
All changed to SignIn now!

seiyaterada reviewed (approved):
Looks really good, excited to see what it will look like! Just one small nit-picky detail that I touched on.

BraidenCutforth committed:
PAS-51

- Change Login to Sign In
- Fix styling with Sign In page
- Modify TextButton to accept style overrides
- Modify text input sizing to be controlled by parent

isaiahdoyle commented on src/App.tsx:
```diff
@@ -1,27 +1,9 @@
-			</header>
-		</div>
-	);
+	return <Pages />;
```
I'm excited for this :)

isaiahdoyle reviewed (approved):
Nice!
